src/sentry/replays/query.py

```diff
@@ -371,6 +371,108 @@ def _activity_score():
     )
 
 
+def _exp_event_ids_agg(alias, ids_type_list):
+    id_types_to_aggregate = []
+    for id_type in ids_type_list:
+        id_types_to_aggregate.append(
+            Function(
```

Comment:
This could be abstracted and made re-usable by the other functions in this module.